api/gateway.proto

```diff
@@ -432,3 +432,8 @@ message GatewayConnectionStats {
 
   // next: 13
 }
+
+message GatewayConnectionStatsEntry{
+  GatewayIdentifiers gateway_ids = 1;
+  GatewayConnectionStats gateway_connection_stats = 2;
```

Comment:
Can't we do a `map<string, GatewayConnectionStats>` with UID as key? Or is this better this way?

Reply:
Yup this ends up being cleaner.